src/components/Register/Register.jsx

```diff
@@ -0,0 +1,225 @@
+          <li>
+            <input
+              type="text"
+              name="retailerName"
+              onChange={(e) => setRetailerName(e.target.value)}
+              placeholder="Store Name"
+            />
+          </li>
+          <li>
+            <input
+              type="text"
+              name="username"
+              onChange={(e) => setUsername(e.target.value)}
+              placeholder="Username"
+            />
+          </li>
+          <li>
+            <input
+              type="password"
+              name="password"
+              onChange={(e) => setPassword(e.target.value)}
+              placeholder="Password"
+            />
+          </li>
+          <li>
+            <input
+              type="password"
+              name="password"
+              onChange={(e) => setConfirmPassword(e.target.value)}
+              placeholder="Confirm Password"
+            />
+          </li>
+          <li>
+            <input
+              type="text"
+              name="address"
+              onChange={(e) => setAddress(e.target.value)}
+              placeholder="Address"
+            />
+          </li>
+          <li>
+            <input
+              type="text"
+              name="city"
+              onChange={(e) => setCity(e.target.value)}
+              placeholder="City"
+            />
+            <select
+              name="state"
+              id="state"
+              onChange={(e) => setState(e.target.value)}
+            >
+              {states.map((state) => {
+                return (
+                  <option key={state} value={state}>
+                    {state}
+                  </option>
+                );
+              })}
+            </select>
+            <input
+              type="text"
+              name="zipCode"
+              onChange={(e) => setZipcode(e.target.value)}
+              placeholder="Zip Code"
+            />
+          </li>
+          <li>
+            <input
+              type="text"
+              name="phoneNumber"
+              onChange={(e) => setPhoneNumber(e.target.value)}
+              placeholder="Phone Number"
+            />
+          </li>
+          <li>
+            Opening Time:{" "}
+            <input
+              type="time"
+              name="openingTime"
+              onChange={(e) => setOpeningTime(e.target.value)}
+              placeholder="Opening Time"
+            />
+            Closing Time:{" "}
+            <input
+              type="time"
+              name="closingTime"
+              onChange={(e) => setClosingTime(e.target.value)}
+              placeholder="Closing Time"
+            />
+          </li>
+          <li>
+            <input
+              type="number"
+              name="maxCapacity"
+              onChange={(e) => setMaxCapacity(e.target.value)}
+              placeholder="Max Capacity"
+            />
+          </li>
+        </ul>
+        <button type="submit">Register</button>
```

Comment:
Might need to add onClick={() => props.setIsLogin(true)} to the submit button so it goes to the login component after registering

Reply:
you're right!  I'll add that in